velox/functions/prestosql/window/AggregateWindow.cpp

```diff
@@ -155,21 +164,45 @@ class AggregateWindowFunction : public exec::WindowFunction {
     // computation for each row needs the values from its frameStart to frameEnd
     // indices. So for each apply() we read arguments from the least
     // frameStart to the highest frameEnd value in the row block.
-    vector_size_t firstRow = rawFrameStarts[0];
-    vector_size_t lastRow = rawFrameEnds[0];
+    vector_size_t i = 0;
```

Comment:
https://github.com/facebookincubator/velox/pull/3200 is submitted now. Please can you rebase.